app/main.py

```diff
@@ -1,8 +1,42 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: float, clean_mark: float, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: float,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class *
+                 (self.clean_power - car.clean_mark) *
+                 self.average_rating /
+                 self.distance_from_city_center)
```

Comment:
The calculation of the washing price could lead to unexpected results due to the order of operations. Consider using parentheses to make the intended order explicit, especially around the division and multiplication operations.